sentry_sdk/client.py

```diff
@@ -227,6 +242,55 @@ def _should_capture(
 
         return True
 
+    def _update_session_from_event(
+        self,
+        session,  # type: Session
+        event,  # type: Event
+    ):
+        # type: (...) -> None
+
+        crashed = False
+        did = None
+        ip_address = None
+        user_agent = None
+        error_inc = 0
+
+        # Figure out if this counts as an error and if we should mark the
+        # session as crashed.
+        level = event.get("level")
+        if level == "fatal":
```

Comment:
I'd move this logic to another function. Can be tested in isolation and might end up duplicated. Could it be in SentryEvent? `event.is_crashed()`?

Reply:
There is no sentry event object, so there is no abstraction we can use here. Will keep it here for now.